docs/scripts/cosmwasm/init.sh

```diff
@@ -8,7 +8,7 @@ STAKING_AMOUNT=1000000000uwasm
 CHAINFLAG="--chain-id ${CHAIN_ID}"
 TXFLAG="--chain-id ${CHAIN_ID} --gas-prices 0uwasm --gas auto --gas-adjustment 1.3"
 
-NAMESPACE_ID=$(echo $RANDOM | md5sum | head -c 16; echo;)
+NAMESPACE_ID=$(openssl rand -hex 8)
```

Comment:
i guess `openssl` exists by default? if not probably `brew install`?

Reply:
for most users who will run a rollup, i believe so. then for macs this removes the dependency of brew as well

it works after i install dependencies as we have in celestia docs, just not on a completely blank machine (testing out in docker)
```
docker run -it --entrypoint "/bin/bash" ubuntu:20.04
apt update
apt upgrade
apt install curl tar wget clang pkg-config libssl-dev jq build-essential git make ncdu -y
```

it was installed with libssl-dev

this worked, i feel like most people running a rollup will have this installed already tbh
```
docker run -it --entrypoint "/bin/bash" ubuntu:20.04
apt update
apt upgrade
apt install openssl
```

Reply:
so, it should be installed w dependency installations